src/Markup/Avalonia.Markup.Xaml.Loader/CompilerExtensions/AvaloniaXamlIlLanguage.cs

```diff
@@ -366,5 +376,43 @@ public IXamlCustomAttribute GetCustomAttribute(IXamlProperty property, IXamlType
             result = null;
             return false;
         }
+
+        private static bool ConvertDefinitionList(
```

Comment:
We should probably move intrinsic converters away from AvaloniaXamlIlLanguage.cs, it's getting too fat